sink-connector-lightweight/docker/docker.env

```diff
@@ -2,10 +2,10 @@ export database.hostname="mysql-master"
 export database.port="3306"
 export database.user="root"
 export database.password="root"
-export database.include.list=sbtest
+export database.include.list=test
```

Comment:
Also please rollback this.

Reply:
Yes, at the moment I am testing runners, I will remove all discrepancies with the main project, except for my tests when everything is ready.